include/quicer_types.hrl

```diff
@@ -271,20 +275,37 @@
                             , alpns := string() | undefined
                             }.
 
--type conn_closed_props() :: map().
+-type conn_closed_props() :: #{ is_handshake_completed := boolean()
+                              , is_peer_acked := boolean()
+                              , is_app_closing := boolean()
+                              }.
 
--type transport_shutdown_info() :: #{ is_conn_shutdown := boolean()
-                                    , is_app_closing := boolean()
-                                    , is_shutdown_by_app := boolean()
-                                    , is_closed_remotely := boolean()
-                                    , status := atom_reason()
-                                    , error := error_code()
-                                    }.
+-type transport_shutdown_props() :: #{ is_conn_shutdown := boolean()
+                                     , is_app_closing := boolean()
+                                     , is_shutdown_by_app := boolean()
+                                     , is_closed_remotely := boolean()
+                                     , status := atom_reason()
+                                     , error := error_code()
+                                     }.
 
 %% Stream Event Props
--type stream_start_completed_props() :: map().
+-type stream_start_completed_props() :: #{ status := atom()
+                                         , stream_id := integer()
+                                         , is_peer_accepted := boolean() }.
```

Comment:
Codestyle: add \n before `}` to make it consistent with the test of the typespecs.